interfaces/udev/backend.go

```diff
@@ -55,16 +62,43 @@ func (b *Backend) Setup(snapInfo *snap.Info, devMode bool, repo *interfaces.Repo
 	if err != nil {
 		return fmt.Errorf("cannot obtain udev security snippets for snap %q: %s", snapName, err)
 	}
-	content, err := b.combineSnippets(snapInfo, snippets)
+	combinedSnippets, err := b.combineSnippets(snapInfo, snippets)
```

Comment:
Let's please keep this variable named `content` as that's what is used everywhere else around this concept.